### ISSUE TITLE
False negative on function call parameter alignment

### ISSUE THREAD

**Issue by hktonylee** [progre/tslint-config-airbnb#66]
In [7.15](https://github.com/airbnb/javascript) the example shows that we need to align this way:

```
// good
function foo(
  bar,
  baz,
  quux,
) {
  // ...
}
```

and forbid this way
```
// bad
function foo(bar,
             baz,
             quux) {
  // ...
}
```

However the current tslint config does not report error in this example:

```
function test(a: any,
              ...args: any[]) {
  console.log(a);
}

const bbs = test(123,
                 456);
```

And now it even accepts the following intendation, which looks weird:

```
const bba = test(
          body.fieldA,
          body.fieldB,
          body.fieldC,
          body.fieldD,
          () => {
            console.log(1);
          },
    );
```

The current version I am using is ^5.11.1
